cli/src/mdastToRst.ts

```diff
@@ -238,6 +238,11 @@ const visitors: {
     if (value === undefined) {
       return;
     }
+    if (value.endsWith(" - ")) {
+      // rst does not like the "- ``" character sequence. Don't ask me why.
```

Comment:
How did you discover this? Also does this lose any information from the snippet if we cut off the hyphen?

Reply:
it does not -- it looks like the format `parameter - `definition is a javadoc thing that just happens to break rst in bulleted lists. You can try it here to verify i'm not crazy: http://rst.ninjs.org/#LSBpIGFtIGEgZG9nCi0gZG9nIC0gZG9nCi0gYGBkb2dgYCBkb2cKLSBgYGRvZyAtIGBgIGRvZwotIGBgZG9nIGBgIGRvZwotIGBgZG9nYGA6Y2F0